### DISCUSSION TITLE
What video file formats are accepted by LabGym?

### DISCUSSION THREAD

**Discussion by CENICOL** [umyelab/LabGym#213] [Q&A]
Which file formats are compatible with LabGym? Are there preferred file formats for use on this platform? I currently have .mp4 files but there are multiple types of mp4 so I want to be sure that mine is compatible with the program.

**Comment by yujiahu415** [comment]
Hi,
LabGym accepts most of the commonly used video formats. The only one that cannot work is .WMV. MP4 format should work well. There's no preference on the format.

**Reply by andmyers** [reply]
Will .mkv files work, or would it be better to batch convert to .avi or .mp4?

**Reply by yujiahu415** [reply]
mkv should also work. No need to convert to avi or mp4.
